README.md

```diff
@@ -15,21 +15,21 @@ Please read [Environment.md](docs/dev/Environment.md) for setting up and running
 ## More Information
 Please see the [Apache Drill Website](http://drill.apache.org/) or the [Apache Drill Documentation](http://drill.apache.org/docs/) for more information including:
 
- * Remote Execution Installation Instructions
- * [Running Drill on Docker instructions](https://drill.apache.org/docs/running-drill-on-docker/)
- * Information about how to submit logical and distributed physical plans
- * More example queries and sample data
- * Find out ways to be involved or discuss Drill
+* Remote Execution Installation Instructions
+* [Running Drill on Docker instructions](docs/dev/Docker.md)
```

Comment:
Initially, the link was correct, but for some reason, it became broken... Could you please instead fix the page for the Drill website? Here is the corresponding doc: https://github.com/apache/drill/blob/gh-pages/_docs/install/installing-drill-in-embedded-mode/011-running-drill-on-docker.md

We shouldn't use the relative link here since this README.md is also displayed in https://hub.docker.com/r/apache/drill and this link will be broken there.

Reply:
Ok gotcha, this seems  to be an issue related to another branch, so I'll have to close this ticket and then make another pr for the gh-pages